### ISSUE TITLE
read SQL Tables more gracefully

### ISSUE THREAD

**Issue by NiklasPhabian** [SpatioTemporal/STAREPandas#16]
probably just a minor tweak. Pandas gives us almost all we need already:

```python3
engine = sqlalchemy.create_engine(uri)
df = pandas.read_sql_table(table_name, con=engine)
```
Now we need to convert the blob in the stare column to a numpy array

```python3
df.stare_cover = df.stare_cover.apply(func=numpy.frombuffer, args=('int64',))
``` 

Should probably wrap this up

**Comment by NiklasPhabian** [comment]
... well; of coures this is only the case if we stare sets of SIDs!

**Comment by NiklasPhabian** [comment]
solved in 6bdc9934d00961ae15061f9b98efbb7ab22fd8f4
